integration/functions.sh

```diff
@@ -126,14 +126,19 @@ log_prep_test() {
     local logsigner_opts="--force_master"
   fi
 
+  if [[ "${WITH_PKCS11}" == "true" ]]; then
+    export SOFTHSM_CONF=${PWD}/testdata/softhsm.conf
```

Comment:
Should document that `$SOFTHSM_CONF` is now populated by this function (there's a list of environment variables populated at the top of the function).